doc/install.rst

```diff
@@ -152,8 +161,8 @@ or from PyGMT's `GitHub repository <https://github.com/GenericMappingTools/pygmt
 
     pip install git+https://github.com/GenericMappingTools/pygmt.git#egg=pygmt
 
-Any of the above methods (conda/pip) should allow you to use the ``pygmt``
-library from Python.
+Any of the above methods (conda/pip) should allow you to use the PyGMT library
+from Python.
```

Comment:
I'm wondering should we call PyGMT a "library" or a "package".

Reply:
I think it is a package.